app/modules/conversations/conversations_router.py

```
@@ -222,4 +225,4 @@ async def remove_access(
         )
         return {"message": "Access removed successfully"}
     except ShareChatServiceError as e:
-        raise HTTPException(status_code=400, detail=str(e))
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)